Improved the README (again) (#337)
openandroidinstaller-dev/openandroidinstaller: README.md

> This application is currently in beta state, so use at your own risk! While many people tested the application so far and we heard of no bricked devices, things might still go wrong.

> [!IMPORTANT]
> **Unlocking the bootloader will erase all data on your device!**
> This also includes your DRM keys, which are stored in the Trim Area partition (also called TA) in case your device is fairly recent and supports DRM L1. Those devices will be downgraded to DRM L3. Devices on DRM L3 by default will not be affected.
> Before proceeding, ensure the data you would like to retain is backed up to your PC and/or your Google account, or equivalent. Please note that OEM backup solutions like Samsung and Motorola backup may not be accessible from LineageOS once installed.
> If you wish to backup the TA partition first, you can find tutorials related to your device on the internet.

## Usage

> Linux is currently the best supported platform (tested with Ubuntu 20.04/22.04 LTS). Windows and macOS are also supported, but you might experience more issues.<br />

Reviewer: I don't like the linebreak here, I think it adds not too much.

So far there is no support for ARM-based systems.

### 1. Download OpenAndroidInstaller

> Download the correct version for your OS [from the Releases](https://github.com/openandroidinstaller-dev/openandroidinstaller/releases).

Reviewer: There are versions for Linux, Windows and MacOS (without ARM, maybe this can go here). Also the "(You might need to change permissions to run the executable)" needs to be here.

> For Linux users, you can get the [official Flatpak version from Flathub](https://flathub.org/apps/org.openandroidinstaller.OpenAndroidInstaller). (You might need to change permissions to run the executable)

Reviewer: ```suggestion
For Linux users, you can also get the [official Flatpak version from Flathub](https://flathub.org/apps/org.openandroidinstaller.OpenAndroidInstaller).
```

> For Windows users, you might need to [install the Universal USB Drivers](https://adb.clockworkmod.com) and other potentially drivers needed for your device.

Reviewer: Maybe this whole section could be structured by OS? Then the information needed for each OS can be grouped together properly?

Author: I like the idea, will implement it soon

### 2. Download the custom ROM image & the TWRP recovery image (& optionally addons)

Here are the official links for:

  - some custom ROMs:
    - [LineageOS](https://wiki.lineageos.org/devices)
    - [/e/OS](https://doc.e.foundation/devices)
    - [LineageOS for MicroG](https://download.lineage.microg.org)
    - [BlissRoms](https://blissroms.org)
    - [PixelExperience](https://download.pixelexperience.org)
  - TWRP Recovery:
    - [TWRP recovery](https://twrp.me/Devices)
  - Optional Addons:
    - There are different packages of *Google Apps* available:
      - [MindTheGapps](https://wiki.lineageos.org/gapps#downloads)
      - [NikGApps](https://nikgapps.com)
    - [MicroG](https://microg.org)
      - The recommended way to install MicroG is to use the zip file provided here: [https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases](https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases).
    - [F-Droid App-Store](https://f-droid.org/en/packages/org.fdroid.fdroid.privileged.ota).

>   - or you can just search the web or the [XDA Developers forum](https://forum.xda-developers.com) for an appropriate version for your device.

Reviewer 2: The XDA Forums URL has changed to `https://xdaforums.com`

Author: Fixed

> [!IMPORTANT]
> **Unlocking the bootloader will erase all data on your device!**
> This also includes your DRM keys, which are stored in the Trim Area partition (also called TA) in case your device is fairly recent and supports DRM L1. Those devices will be downgraded to DRM L3. Devices on DRM L3 by default will not be affected.
> Depending on your device you might be able to back up the TA partition using exploits and gaining temporary root access. On Sony Xperia 1/5 series phones DRM L1 will return once the bootloader is relocked.
> Before proceeding, ensure the data you would like to retain is backed up to your PC and/or your Google account, or equivalent. Please note that OEM backup solutions like Samsung and Motorola backup may not be accessible from LineageOS once installed.

## Usage

Linux is currently the best supported platform (tested with Ubuntu 20.04/22.04 LTS). Windows and macOS are also supported, but you might experience more issues.

### 1. Download OpenAndroidInstaller

> [!WARNING]

> > ARM-based systems are **unsupported**.

Reviewer 2: `unsupported` does not sound right, maybe `not supported` would be better and amplify the unsupported status?

> Follow the instructions for your desktop OS. The executables are compressed inside `.zip` files, so you'll have to extract them first (make sure you have extracting software installed). If you get prompted to, you'll have to adjust the permission of the executable to ensure its proper functionality. 

Reviewer 2: I don't think, that `desktop OS` needs to be specified, just `OS` should be alright...

#### Linux

You can download OAI from either:

> 1. From Flathub: [`org.openandroidinstaller.OpenAndroidInstaller`](https://flathub.org/apps/org.openandroidinstaller.OpenAndroidInstaller) (recommended)

Reviewer 2: `From` is double worded, as it would read like "You can download OAI from either from Flathub [...]" and should be therefore removed.
I think the `either` should also be removed and the download options would better be in a unsorted list.

2. GitHub Releases: [`openandroidinstaller-ubuntu-latest.zip`](https://github.com/openandroidinstaller-dev/openandroidinstaller/releases/download/latest/openandroidinstaller-ubuntu-latest.zip)

#### Windows

You can download OAI for Windows from the GitHub Releases: [`openandroidinstaller-windows.zip`](https://github.com/openandroidinstaller-dev/openandroidinstaller/releases/download/latest/openandroidinstaller-windows.zip)

You might also need to [install the Universal USB Drivers](https://adb.clockworkmod.com) and other potentially drivers needed for your device.

#### macOS

For macOS, you can get OpenAndroidInstaller from the GitHub Releases: [`openandroidinstaller-macos.zip`](https://github.com/openandroidinstaller-dev/openandroidinstaller/releases/download/latest/openandroidinstaller-macos.zip)

> ### 2. Download the custom ROM image & the TWRP recovery image (& optionally addons)

Reviewer 2: I would rephrase the header like this: `Download the custom ROM, recovery image & optional addons`, as support for multiple recoveries is coming up.

Here are the official links for:

>   - some custom ROMs:

Reviewer 2: [GrapheneOS](https://grapheneos.org), [CalyxOS](https://calyxos.org) and [DivestOS](https://divestos.org) should also be added

    - [LineageOS](https://wiki.lineageos.org/devices)
    - [/e/OS](https://doc.e.foundation/devices)
    - [LineageOS for MicroG](https://download.lineage.microg.org)
    - [BlissRoms](https://blissroms.org)
    - [PixelExperience](https://download.pixelexperience.org)

>   - TWRP Recovery:

Reviewer 2: This should just be named `Recovery`

    - [TWRP recovery](https://twrp.me/Devices)
  - Optional Addons:
    - There are different packages of *Google Apps* available:
      - [MindTheGapps](https://wiki.lineageos.org/gapps#downloads)
      - [NikGApps](https://nikgapps.com)
    - [MicroG](https://microg.org)
      - The recommended way to install MicroG is from the `.zip` file provided here: [`MinMicroG-abuse-CI/releases`](https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases).
    - [F-Droid App-Store](https://f-droid.org/en/packages/org.fdroid.fdroid.privileged.ota).
  - or you can just search the web or the [XDA Developers forum](https://xdaforums.com) to find an appropriate version for your device.

> ### 3. Start the desktop app

Reviewer 2: I think `Start OpenAndroidInstaller` would fit better...

> There, you will be prompted to plug your device to your computer and you will be given a step-by-step tutorial on how to install the ROM, TWRP and the addons for your specific model.

Reviewer 2: Just a rephrasing including a grammar correction:
`After starting the app you will be prompted to plug your device into your computer and you will be given a step-by-step tutorial on how to install your new OS for your specific device.`

> This application is currently in beta state, so use at your own risk! While many people tested the application so far and we heard of no bricked devices, things might still go wrong.

> [!IMPORTANT]
> **Unlocking the bootloader will erase all data on your device!**
> This also includes your DRM keys, which are stored in the Trim Area partition (also called TA) in case your device is fairly recent and supports DRM L1. Those devices will be downgraded to DRM L3. Devices on DRM L3 by default will not be affected.
> Depending on your device you might be able to back up the TA partition using exploits and gaining temporary root access. On Sony Xperia 1/5 series phones DRM L1 will return once the bootloader is relocked.
> Before proceeding, ensure the data you would like to retain is backed up to your PC and/or your Google account, or equivalent. Please note that OEM backup solutions like Samsung and Motorola backup may not be accessible from LineageOS once installed.

## Usage

Linux is currently the best supported platform (tested with Ubuntu 20.04/22.04 LTS). Windows and macOS are also supported, but you might experience more issues.

> ### 1. Download OpenAndroidInstaller

Author: Alright, I'll fix these 2 things you have pointed out and this should be ready to be merged
